apis/apis-overview/budget-api.md

```diff
@@ -1,8 +1,8 @@
-# Budget API
+# Budget API (v2)
 
-The Budget API allows you to create, update, and delete recurring budget rules to control your Kubernetes spending. Weekly and monthly budgets can be established to set limits on cost spend.
+The Budget API allows you to create, update, and delete recurring budget rules to control your Kubernetes spending. Weekly and monthly budgets can be established to set limits on cost spend, with the option to configure alerts for reaching specified budget thresholds via email, Slack, or Microsoft Teams.
```

Comment:
Can we name it actions instead of alerts as that the word used.

Reply:
I know Alerts is an existing feature, but saying the user can configure "an action for email" doesn't exactly convey what is happening here. We elaborate later in the article how this is done through the actions param